metadata-transformer/src/main/java/no/unit/nva/metadata/DcTerms.java

```diff
@@ -64,20 +65,30 @@
     VALID("valid");
 
     private static final String DCTERMS_PREFIX = "http://purl.org/dc/terms/";
+    public static final String DC = "dc.";
+    public static final String DCTERMS = "dcterms.";
+    public static final String EMPTY_STRING = "";
 
     private final String localName;
 
     DcTerms(String localName) {
         this.localName = localName;
     }
 
+    @Override
     public IRI getIri(ValueFactory valueFactory) {
         return valueFactory.createIRI(DCTERMS_PREFIX, localName);
     }
 
-    public static Optional<DcTerms> getTermByValue(String term) {
+    public static Optional<DcTerms> getTermByValue(String candidate) {
+        String term = replaceInternalPrefixes(candidate);
         return Optional.of(Arrays.stream(values())
                 .filter(dcTerm -> dcTerm.localName.equalsIgnoreCase(term))
                 .findAny()).orElse(Optional.empty());
     }
+
+    private static String replaceInternalPrefixes(String input) {
```

Comment:
This removes the need for special treatment of dc/dcterms as dc is in practice a superset of dcterms.